packages/use-sync-external-store/src/__tests__/useSyncExternalStoreShared-test.js

```diff
@@ -857,8 +851,6 @@ describe('Shared useSyncExternalStore behavior (shim and built-in)', () => {
     expect(container.textContent).toEqual('UPDATED');
   });
 
-  // The selector implementation uses the lazy ref initialization pattern
-  // @gate !(enableUseRefAccessWarning && __DEV__)
```

Comment:
See example this was opted out, because it supposedly uses the "lazy ref init pattern", so the checks that try to allow that are failing here.